storage.bs

```diff
@@ -129,12 +129,14 @@ either the <a>obtain a local storage bottle map</a> or the
 <p class=note>If you are defining a standard for such an API, consider filing an issue against this
 standard for assistance and review.
 
-<p>To isolate this data this standard defines a <a for=/>storage shed</a> which segments
-<a>storage shelves</a> by a <a>storage key</a>. A <a>storage shelf</a> in turn consists of a
-<a>storage bucket</a> and will likely consist of multiple <a>storage buckets</a> in the future to
-allow for different storage policies. And lastly, a <a>storage bucket</a> consists of
+<p id=model-description>To isolate this data this standard defines a <a for=/>storage shed</a>
+which segments <a>storage shelves</a> by a <a>storage key</a>. A <a>storage shelf</a> in turn
+consists of a <a>storage bucket</a> and will likely consist of multiple <a>storage buckets</a> in
+the future to allow for different storage policies. And lastly, a <a>storage bucket</a> consists of
 <a>storage bottles</a>, one for each <a>storage endpoint</a>.
 
+<p><img src=assets/model-diagram.svg alt="Storage model diagram" aria-describedBy=model-description
+width=396 height=815>
```

Comment:
Is this really preferable over `alt="Storage model diagram (described above)."` and no `aria-describedby` (lowercase please)?

Flipping the order and using "described below" might be even better as then the image serves as a kind of heading?